terraso_backend/apps/audit_logs/models.py

```diff
@@ -0,0 +1,50 @@
+from django.conf import settings
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+# Create your models here.
+CREATE = 1
+READ = 2
+CHANGE = 3
+DELETE = 4
+
+EVENT_CHOICES = (
+    (CREATE, _("CREATE")),
+    (READ, _("READ")),
+    (CHANGE, _("CHANGE")),
+    (DELETE, _("DELETE")),
+)
+
+
+class Log(models.Model):
+    """
+    Log model for audits logs
+    """
+    timestamp = models.DateTimeField(auto_now_add=True)
+    client_timestamp = models.DateTimeField()
+    # TODO - Discuss: if we should delete on cascade or not
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        verbose_name='user'
+    )
+    resource_id = models.TextField()
```

Comment:
This is what I talked about with you guys; I did this to avoid generic foreign keys but generic foreign keys can be as well set here. Is essentially the same behind the scenes.